### ISSUE TITLE
NAACL-2019-Structured Minimally Supervised Learning for Neural Relation Extraction

### ISSUE THREAD

**Issue by BrambleXu** [BrambleXu/knowledge-graph-learning#96]
**一句话总结：**

基于SPENs， 考虑了the inclusion of latent variables with deterministic factors。

讲不清楚，不建议看。

**资源：**

- [pdf](<https://arxiv.org/abs/1904.00118>)
- [code](<https://github.com/bflashcp3f/PCNN-NMAR>): python2.7, pytorch0.4.1
- [paper-with-code](<https://paperswithcode.com/paper/structured-minimally-supervised-learning-for>)

**关键字：**

- dataset: NYTFB
- DS noisy
- combines the benefits of learned representations and structured learning
- explicitly reasoning about missing data during learning, our approach enables large-scale
  training of 1D convolutional neural networks while mitigating the issue of label noise inherent in distant supervision

**笔记：**

Recent work has shown promising results from large-scale training of neural networks
for relation extraction (Toutanova et al., 2015; Zeng et al., 2015).

1D CNN are used to define potentials over latent relation mention variables. These mention-level variables are related to observed facts in a KB using a set of deterministic factors, followed by pairwise potentials that encourage agreement between extracted propositions and observed facts, but also enable inference to override these soft constraints during learning, allowing for the possibility of missing information

这个工作和Structured Prediction Energy Networks (SPENs) (Belanger and McCallum, 2016)相关，the key differences are the application to minimally supervised relation extraction (as opposed to multi-label classification) and the inclusion of latent variables with deterministic factors, which we demonstrate enables effective learning in the presence of missing data in distant supervision

**模型图：**

**结果**：

![image-20190408095654764](https://ws3.sinaimg.cn/large/006tNc79gy1g1uy0po1cij30rd09v41a.jpg)
